### PR TITLE
consolidate coralogix schema

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

File: src/coralogix/schema.js [hunk 1]
```diff
@@ -54,71 +55,79 @@ const schema = {
...
-        id: str(vcl`if(req.http.X-CDN-Request-ID, req.http.X-CDN-Request-ID, randomstr(8, "0123456789abcdef") + "-" + randomstr(4, "0123456789abcdef") + "-" + randomstr(4, "0123456789abcdef") + "-" + randomstr(1, "89ab") + randomstr(3, "0123456789abcdef") + "-" + randomstr(12, "0123456789abcdef"))`),
-        method: str('%m'),
-        protocol: str(vcl`if(fastly_info.is_h2, "HTTP/2", "HTTP/1.1")`),
-        h2: vcl`if(fastly_info.is_h2, "true", "false")`,
-        h2_push: vcl`if(fastly_info.h2.is_push, "true", "false")`,
-        is_ipv6: vcl`if(req.is_ipv6, "true", "false")`,
-        url: str(vcl`cstr_escape(if(req.http.X-Orig-Url, req.http.X-Orig-Url, req.url))`),
+      city_name: str(vcl`client.geo.city.ascii`),
+      country_name: str(vcl`client.geo.country_name.ascii`),
+      connection_speed: str(vcl`client.geo.conn_speed`),
```
Comment (trieloff) [inline src/coralogix/schema.js:80]: I don't think this is useful, we can as well remove it

**Review by trieloff — APPROVED** [review]

File: src/coralogix/schema.js (hunk 1 above)
Comment (stefan-guggisberg) [inline src/coralogix/schema.js:80]:
```suggestion
```

Comment (stefan-guggisberg) [reply src/coralogix/schema.js]: good point , done

**Pull request merged**